solutions.txt

```diff
@@ -116,13 +111,7 @@ Exercise 10:
 */
 
 // Complete Exercise 10 below...
-let hassoup = ['soup'];
-         
-            if( soup ) {
-               document.write('has soup');
-            } else {
-               document.write('doesnothave soup');
-
+var hasSoup = !(-1 === foods.indexOf('soup'));
```

Comment:
nice job putting together all these ideas! the `!` for `not`, the boolean expression, and the use of `indexOf()` 👍